versatile-api/src/main/java/com/fb/versatile_api/HotelController.java

```diff
@@ -0,0 +1,70 @@
+package com.fb.versatile_api;
+
+import java.util.List;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class HotelController {
+
+    private Hotel hotel = new Hotel();
+
+    @GetMapping("/hotel")
+    public String getHotelInfo() {
```

Comment:
Let's change outputs to JSON. Return `Hotel` instead of `String`. If you pass `Content-Type` - `Application/json` as one of the request header from you postman, your controller will automatically convert your object to JSON